components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -125,9 +132,46 @@ void DeletePromoCodeFile(const base::FilePath& promo_code_file) {
   }
 }
 
+#if BUILDFLAG(IS_MAC)
+std::string ReadPromoCodeFromXattr() {
+  static constexpr char kRefCodeAttr[] = "com.brave.refcode";
+  base::FilePath bundle_path = base::apple::OuterBundlePath();
+  std::vector<char> value;
+  int result_errno =
+      brave::GetFileExtendedAttribute(bundle_path, kRefCodeAttr, &value);
+  if (result_errno == ENOATTR) {
+    VLOG(0) << "Could not get promo code from " << bundle_path
+            << ". The extended attribute " << kRefCodeAttr << " was not found.";
+  } else if (result_errno != 0) {
+    LOG(ERROR) << "Could not get promo code from " << bundle_path
```

Comment:
sorry, I mean all of these should be VLOG

Reply:
Fixed in [18fa76b](https://github.com/brave/brave-core/commit/18fa76b2fafc403c44d02bc93875a70814f4c40f)